Node.js/sampleresponse.js

```diff
@@ -101,13 +109,19 @@ function poisoningMessageHandler(buf,md) {
 }
 
 function sendToQueue(buf, md, queue) {
-
   return mqBoilerPlate.openMQReplyToConnection(queue, 'DYNREP')
+  .then(() => {
```

Comment:
Add comment explaining why this is necessary.

Reply:
This is incorrect. The suspend in in the response, and is on the async response get callback.

Reply:
No, the 1st async call isn't accessing the reply to queue, it is accessing the get queue. 

An async callback thread can't invoke an async MQ call.